src/RF24Logger.h

```diff
@@ -22,6 +22,13 @@
 #include "RF24LogLevel.h"
 #include "RF24LogHandler.h"
 
+#if defined (ARDUINO_ARCH_AVR)
+  #define RF24LOG_FLASHIFY(A) F(A)
+  #define RF24LOGGER_info(vendorId, message, ...) (rf24Logger.log(RF24LogLevel::INFO, RF24LOG_FLASHIFY(vendorId), RF24LOG_FLASHIFY(message), ##__VA_ARGS__))
+#elif

```

Comment:
I think this line should be `#else`. The Arduino CI action is throwing an error about not using a condition following the `#elif` on this line.

Reply:
Yes, you are right. I already fixed this and pushed the changes.